services/forms-api/openapi.yml

```diff
@@ -0,0 +1,530 @@
+
+    post:
+      tags:
+        - Forms
+      summary: Creates form.
+      description: This endpoint creates a new form-builder form entry in the database.
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Form'
+      responses:
+        '201':
+          description: OK.
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/GetFormResponse'
+        '400':
+          description: Failed validating request body.
+          content:
+            'application/json':
+              schema:
+                $ref: '../common-openapi.yml#/components/schemas/jsonApiError'
+        '403':
+          $ref: '../common-openapi.yml#/components/responses/missingApiKeyResponse'
+
+  /forms/{formId}:
+    get:
+      tags:
+        - Forms
+      parameters:
+        - name: formId
+          in: path
+          required: true
+          schema:
+            type: string
+      summary: Get form.
+      description: This endpoint returns a form-builder form by its unique id.
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          description: OK.
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/GetFormResponse'
+        '403':
+          $ref: '../common-openapi.yml#/components/responses/missingApiKeyResponse'
+        '404':
+          description: Form with id not found.
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NoFormFoundResponse'
+
+    put:
+      tags:
+        - Forms
+      parameters:
+        - name: formId
+          in: path
+          required: true
+          schema:
+            type: string
+      summary: Updates or creates a form.
+      description: |
+        This endpoint updates a current form. If the form does not exist, it will be created.
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Form'
+      responses:
+        '200':
+          description: OK.
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/GetFormResponse'
+        '403':
+          $ref: '../common-openapi.yml#/components/responses/missingApiKeyResponse'
+
+    delete:
+      tags:
+        - Forms
+      parameters:
+        - name: formId
+          in: path
+          required: true
+          schema:
+            type: string
+      summary: Delete form.
+      description: This endpoint deletes a form from database.
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          description: |
+            OK.
+
+            <b>Note</b> 
```

Comment:
Markdown is supported and should be more preferable here unless there is a reason to use html?

Reply:
fixed